data/templates/trumptable/config.yml

```diff
@@ -0,0 +1,10 @@
+name: The Floor is Trump
+link: https://i.imgur.com/9GaaJ1Q.jpg
```

Comment:
Can you provide a reference URL here? The point is to explain the source of the meme. I don't recognize this one.